### PR TITLE
17 update sbt

### PR DESCRIPTION
**Pull request by mkrzemien**

### CODE REVIEW

File: project/plugins.sbt [hunk 1]
```diff
@@ -1 +1,15 @@
...
+addSbtPlugin("com.softwaremill.sbt-softwaremill" % "sbt-softwaremill-browser-test-js" % sbtSoftwareMillVersion)
+addSbtPlugin("com.typesafe" % "sbt-mima-plugin" % "1.1.0")
+addSbtPlugin("com.typesafe.sbt" % "sbt-twirl" % "1.5.1")
+addSbtPlugin("org.scalameta" % "sbt-mdoc" % "2.3.2")
+addSbtPlugin("com.eed3si9n" % "sbt-projectmatrix" % "0.9.0")
+addSbtPlugin("org.jetbrains.scala" % "sbt-ide-settings" % "1.1.1")
+addSbtPlugin("org.scala-js" % "sbt-scalajs" % "1.10.0")
+addSbtPlugin("io.spray" % "sbt-revolver" % "0.9.1")
+addSbtPlugin("com.eed3si9n" % "sbt-assembly" % "1.2.0")
+addSbtPlugin("io.gatling" % "gatling-sbt" % "4.1.6")
```
Comment (aludwiko) [inline project/plugins.sbt:13]: do we use gatling here?

File: project/plugins.sbt [hunk 2]
```diff
@@ -1 +1,15 @@
-addSbtPlugin("com.jsuereth" % "sbt-pgp" % "1.1.0")
\ No newline at end of file
+val sbtSoftwareMillVersion = "2.0.9"
+addSbtPlugin("com.softwaremill.sbt-softwaremill" % "sbt-softwaremill-common" % sbtSoftwareMillVersion)
+addSbtPlugin("com.softwaremill.sbt-softwaremill" % "sbt-softwaremill-publish" % sbtSoftwareMillVersion)
+addSbtPlugin("com.softwaremill.sbt-softwaremill" % "sbt-softwaremill-browser-test-js" % sbtSoftwareMillVersion)
```
Comment (aludwiko) [inline project/plugins.sbt:4]: required?

**Review by aludwiko — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -42,11 +42,11 @@ Maven:
 
     <dependency>
         <groupId>com.softwaremill.kmq</groupId>
-        <artifactId>core_2.12</artifactId>
+        <artifactId>core_2.13</artifactId>
         <version>0.2</version>
     </dependency>
 
-(Use `core_2.11` if you are using other components depending on Scala 2.11.)
+(Use `core_2.12` if you are using other components depending on Scala 2.12.)
```
Comment (koksyn) [inline README.md:49]: Maybe we should just suggest range of minor versions (to not change it every time, when we upgrade the lib)

E.g.
`(Use `core_2.XX` if you are using other components depending on Scala 2.XX, where XX is a minor version number)`

Comment (mkrzemien) [reply README.md]: I changed kind of the other way around: keeping the specific Scala versions but making the information explicitly about the supported Scala versions. What do you think?

Comment (koksyn) [reply README.md]: It's even better ;)

File: build.sbt [hunk 4]
```diff
@@ -1,82 +1,91 @@
+import com.softwaremill.Publish.ossPublishSettings
+import com.softwaremill.SbtSoftwareMillCommon.commonSmlBuildSettings
+import sbt.Keys._
 import sbt._
-import Keys._
 
 name := "kmq"
+organization := "com.softwaremill.kmq"
+version := "0.3-SNAPSHOT"
```
[the remark this answers is not in the document]
Comment (mkrzemien) [reply build.sbt:8]: OK

**Review by koksyn — APPROVED** [review]

File: project/plugins.sbt (hunk 1 above)
Comment (mkrzemien) [reply project/plugins.sbt:13]: OK, I removed unused plugins.

File: project/plugins.sbt (hunk 2 above)
Comment (mkrzemien) [reply project/plugins.sbt:4]: OK, I removed unused plugins.

**Comment by mkrzemien** [comment]
I`ve created a follow-up issue to update Kafka 2.x to 3.x: https://github.com/softwaremill/kmq/issues/20, because it generates a bunch of additional compilation errors.
Meanwhile, we can still support Scala 2.12.

**Pull request merged**